ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/utils.test.ts

```diff
@@ -125,7 +125,7 @@ describe('utils', () => {
                 includeEpssProbability: false,
                 includeNvdCvss: false,
                 cvesDiscoveredSince: 'START_DATE',
-                cvesDiscoveredStartDate: '2023-10-02',
+                cvesDiscoveredStartDate: '2023-10-02T12:34:56.000Z',
```

Comment:
At 00:00:00 UTC time - this causes a failure in the test when checking against EST.